src/configuration.ts

```diff
@@ -35,7 +35,8 @@ export class Configuration {
             "name": "Fliter Low Confidence",
             "filter_function": filter_low, 
             "confidence_function": get_annotation_confidence, 
-            "mark_deprecated": mark_deprecated
+            "mark_deprecated": mark_deprecated,
+            "default_value": 0.05
```

Comment:
Shouldn't this be 0.5?

Reply:
or is that the user's job to configure?

Reply:
Its the user's job to configure. If no default value is provided then the toolbox item itself has a built in default of 0. 0.05 seems to be closest to the current behavior

Reply:
👍